src/styles/index.scss

```diff
@@ -1,3 +1,141 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, serif;
+}
+
+p {
+  margin: 0;
+}
+
+.card {
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #fff;
+  position: relative;
+  padding: 32px 16px 16px;
+}
+
+.imac-img {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+  display: block;
+}
+
+.name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+
+  // margin: 0;
+  margin-bottom: 4px;
+}
+
+.product-code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  position: relative;
+  color: #616070;
+}
+
+.stars-review {
```

Comment:
The class `.stars-review` does not follow the BEM naming convention. Consider renaming it to something like `.card__stars-review` to maintain consistency.